src/routes/boot.ts

```diff
@@ -530,7 +531,13 @@ const loggedInBoot = async ({
       notifications: { unreadNotificationsCount },
       squads,
       accessToken,
-      exp,
+      exp: {
+        ...exp,
+        a: {
+          ...exp.a,
+          ...(isPlus ? { plus: 1 } : undefined),
```

Comment:
```suggestion
          plus: isPlus ? 1 : undefined
```

its json so undefined will disappear